routes.go

```diff
@@ -161,7 +234,12 @@ func getBranch(w http.ResponseWriter, r *http.Request, p httprouter.Params) {
 		return
 	}
 
-	branch := plumbing.NewRemoteReferenceName("origin", name)
+	actual_remote := p.ByName("forkname")
```

Comment:
Convention in Go is to name variables using "camel case" rather than "snake case". So this would be `actualRemote`.

Reply:
```suggestion
	actualRemote := p.ByName("forkname")
```